onnxruntime/python/tools/quantization/shape_inference.py

```diff
@@ -109,34 +123,45 @@ def quant_pre_process(
                 )
                 logger.error(traceback.format_exc())
 
-            input_model_path = opt_model_path
+            input_model = opt_model_path
 
         if not skip_onnx_shape:
             # ONNX shape inference.
             # According to docs, infer_shapes_path should be used for 2G+ models.
             # If the skip optimization is specified, we could be dealing with a
             # large model. So be on the safe side, save the model
             if model is not None:
-                input_model_path = str(temp_path / "symbolic_shape_inferred.onnx")
+                input_model = str(temp_path / "symbolic_shape_inferred.onnx")
```

Comment:
nit: This is not symbolic shape info, may use a different name like "onnx_shape_inferred.onnx", but this does not impact result.